docs/workforce-management-framework/topics/con-PassportAuth-Module.adoc

```diff
@@ -0,0 +1,41 @@
+[]
+= {WFM-RC-NameShort} Passport.js integration module
+Link to
+link:https://github.com/feedhenry-raincatcher/raincatcher-core/tree/master/cloud/passportauth[passportAuth] module.
+
+{WFM-RC-NameShort} authentication and authorisation module using link:http://passportjs.org[PassportJS].
+The Passport.js module is the default security module for {WFM-RC-NameShort}.
+
+Passport is using "pluggable" authentication plugins called link:http://passportjs.org/docs/configure[strategies].
+Applications can choose which strategies to employ, without creating unnecessary dependencies.
```

Comment:
I think this might be wrong or will cause confusion. To add a new strategy to passport, you need to add a new dependency to raincatcher with the passport strategies npm module.

Reply:
key word is unnecessary . 
but I agree could be confusing